unix/vncpasswd/vncpasswd.cxx

```diff
@@ -109,10 +151,22 @@ static std::vector<uint8_t> readpassword() {
     std::string first = passwd;
     if (first.size() < 6) {
       if (first.empty()) {
-        fprintf(stderr,"Password not changed\n");
+        fprintf(stderr,_("Password not changed\n"));
         exit(1);
       }
-      fprintf(stderr,"Password must be at least 6 characters - try again\n");
+      fprintf(stderr,_("Password must be at least 6 characters - try again\n"));
+      continue;
+    }
+
+    if (first.size() > 8) {
```

Comment:
Nice! But could you split this to a separate commit? It's independent of using pwquality.

Reply:
Perhaps they should be submitted separately

Reply:
A separate commit, yes. But you don't need to open a separate PR for such a small fix.